Update/kage_006.txt

```diff
@@ -1332,7 +1332,7 @@ void main()
 //r今さら時効だと思うが、もしもまどかが家出当時、とある男性の家に転がり込んだことがここでバレようものなら、即刻朝まで説教タイムが開始されるだろう…。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "今さら時効だと思うが、もしもまどかが家出当時、とある男性の家に転がり込んだことがここでバレようものなら、即刻朝まで説教タイムが開始されるだろう…。",
-			NULL, "But Madoka thought the time for that was over, she didn't think that staying over at a man's house was something worth preaching about all morning.", Line_Normal);
+			NULL, "She knew it would be too late by this point, but if it came out now that Madoka had been living with a man when she ran away from home or something, Tomoe would have started lecturing her until the sun rose...", Line_Normal);
```

Comment:
```suggestion
			NULL, "She knew it would be too late by this point, but if it came out now that Madoka had been living with a man when she ran away from home or something, Tomoe would have started lecturing her until the sun came up...", Line_Normal);
```